### ISSUE TITLE
hardcoded DragMarker size

### ISSUE THREAD

**Issue by moovida** [ibrierley/flutter_map_line_editor#11]
Hi, I am trying to make the dragmarker size a user setting, so that one can choose it depending on the device (phone, tablet or pc).
Since I am using containers with decoration to make the handlers, I was experiencing issues with making the marker small (this didn't happen with the icons).

Then I saw that:
```
          DragMarker(
            point: this.points[indexClosure],
            width: 40.0,
            height: 40.0,
```
this width and height are hardcoded. Might this be the issue? 
Is there a reason you chose to put here the size? Wouldn't the size come directly from the child widget? I am sure missign something here. Thanks for any insight.

**Comment by moovida** [comment]
I confirm that it is that value, but am not sure how to handle this, since there is no simple way to make it size-aware in that position.

**Comment by moovida** [comment]
@ibrierley , would it be too ugly for you to add optional size values as:

```
  PolyEditor({
    this.points,
    this.pointIcon,
    this.intermediateIcon,
    this.callbackRefresh,
    this.addClosePathMarker,
    this.pointIconSize = 30,
    this.intermediateIconSize = 30,
  });
```
so that it would result in:
```
      dragMarkers.add(DragMarker(
          point: this.points[indexClosure],
          width: pointIconSize,
          height: pointIconSize,
```

This seems to be working nicely.

**Comment by ian-workcircle** [comment]
Yes, that was an oversight I think, I didn't originally spend a lot of time on this one, and was just trying to get a minimal case up. You could try adding width/height parameters for the PoltEditor class (I guess separate ones for both the point/cornerIcon and the intermediateIcon), and then use them lower down and see how that goes, and naturally feed back if it works ok. I think that would be a sensible addition to the lib, otherwise I'll have a bash when I get chance.

**Comment by ibrierley** [comment]
I was going to write this: 
>>
Yes, that was an oversight I think, I didn't originally spend a lot of time on this one, and was just trying to get a minimal case up. You could try adding width/height parameters for the PoltEditor class (I guess separate ones for both the point/cornerIcon and the intermediateIcon), and then use them lower down and see how that goes, and naturally feed back if it works ok. I think that would be a sensible addition to the lib, otherwise I'll have a bash when I get chance.
>>

So I think that matches up with your post just now. I don't see anything ugly with that at all. Whether we'd want height always be the same as width, I suspect we probably do, but I wouldn't have any complaints if that was passed in as a Size object or separate width/height params

**Comment by moovida** [comment]
Thanks a lot @ibrierley . I have a meeting now, but later I will try the size object option and in case propose you a pull request.

**Comment by moovida** [comment]
This changed has been addressed in PR #12
